src/templates/amfcfg.conf.j2

```diff
@@ -0,0 +1,164 @@
+configuration:
+  amfDBName: {{ amf_database_name }}
+  amfName: AMF
+  debugProfilePort: 5001
+  enableDBStore: false
+  enableSctpLb: false
+  kafkaInfo:
+    brokerPort: 9092
+    brokerUri: sd-core-kafka-headless
+    topicName: sdcore-data-source-amf
+  mongodb:
+    name: {{ default_database_name }}
+    url: {{ database_url }}
+  networkFeatureSupport5GS:
+    emc: 0
+    emcN3: 0
+    emf: 0
+    enable: true
+    imsVoPS: 0
+    iwkN26: 0
+    mcsi: 0
+    mpsi: 0
+  networkName:
+    full: Aether5G
+    short: Aether
+  nfKafka:
+    enable: false
+    topic: sdcore-nf-data-source
+    urls:
+    - sd-core-kafka-headless:9092
+  ngapIpList:
+  - 0.0.0.0
+  ngappPort: 38412
```

Comment:
Should be a variable to pass the `38412` port

Reply:
✅